pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -270,28 +270,32 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication {
     }
 
     /// @inheritdoc IVaultAdmin
-    function collectAggregateFees(address pool) public onlyVaultDelegateCall nonReentrant withRegisteredPool(pool) {
+    function collectAggregateFees(
```

Comment:
I was considering whether we should rename this, as it doesn't by itself collect the fees now - just calculates them. But it is zeroing out the fee balances, so that's a form of collection. I guess the name is fine.

Reply:
well, `swap` doesn't actually do the swap either :)